### ISSUE TITLE
Reproductivity

### ISSUE THREAD

**Issue by hhhhh7733** [OptMN-Lab/sdmgrad#3]
> Hi, thanks for your interest in our work. I just tested our code with the default configuration, and the test mIoU reached 0.43 after only 1 epoch, which is better than the results you reproduced after training for 200 epochs.
> 
> 
> 
> ![image](https://github.com/user-attachments/assets/e4870914-d7c8-4755-aa57-66f63d9f0a97)
> 
> 
> 
> Which model are you using, `segnet` or `mtan`? The default choice should be `mtan`. Additionally, have you made any modifications to the codes? 
> 
>  

 _Originally posted by @bhsimon0810 in [#1](https://github.com/OptMN-Lab/SDMGrad/issues/1#issuecomment-2417774516)_

Hi, when I rerun your code using the default settings without any modifications, I notice a discrepancy between the results shown in this figure and the results I obtain on my side. I am new to this area. Could you please explain why this is happening?

**Comment by bhsimon0810** [comment]
Hi, what results did you get?

**Comment by hhhhh7733** [comment]
> Hi, what results did you get?

<img width="1143" height="26" alt="Image" src="https://github.com/user-attachments/assets/8c54cf00-4549-4a25-9d7b-b9681193ab93" />
Here is the result I got from my side.

**Comment by bhsimon0810** [comment]
Did you use the same dataset? As far as I know, there is both a larger version and a smaller version of the dataset, and the smaller version is preferred. By the way, did you let the experiment run to completion? How did it go?

**Comment by hhhhh7733** [comment]
Yes, I used the smaller dataset, and the experiment ran to completion, but the results weren't particularly good. I think it may be due to different hyperparameters, such as the random seed.
